### DISCUSSION TITLE
What should be the SO tag for Mojo?

### DISCUSSION THREAD

**Discussion by cjohnson318** [modular/modular#344] [General]
It looks like "mojo" is already taken. What about "modular-mojo"?

**Comment by strangemonad** [comment]
do they support emoji's 😈 🔥

**Reply by cjohnson318** [reply]
Probably not. Some tags are prefixed with an icon to show that they're all under one umbrella or another, like AWS. I imagine that they'd like to keep their tagging system focused on ASCII text, for a number of reasons, like, visual design, ease of maintenance, ease of use, etc.

**Reply by strangemonad** [reply]
Sorry, I think my sarcasm was lost in there :)

**Reply by cjohnson318** [reply]
Ha! No worries. 😂
